### DISCUSSION TITLE
Why win-sshproxy.exe & winpath.exe files don't include in MSI distribution 4.1.0?

### DISCUSSION THREAD

**Discussion by imishin1980** [containers/podman#14586] [Q&A] [answered]
I found a solved issue with 4.0.2 and win-sshproxy.exe and winpath.exe files were added to MSI zip. But these files are not included in the latest version?

**Comment by rhatdan** [comment]
@n1hility PTAL

**Comment by n1hility** [comment, marked as answer]
Hi @imishin1980 these are currently only included in the msi, which is our recommended way to install podman for podman machine usage. The zip files only contain the remote client. They could be added to the zip but would require additional setup steps. Is there a use-case where you need to install via zip instead of msi?

**Reply by imishin1980** [reply]
I was installing 4.1.0 podman via MSI and don’t find such files in a bin folder. I don’t find these files even in the remote client 4.1.0 zip. Then I found in the issue list reference regarding 4.0.2 where required files were added to MSI and ZIP. 
If these files are removed from the distribution, why I can’t find additional installation steps to obtain these required files somehow?
Podman will not work after installation without these files do you think this is a good price for distribution size?

**Reply by n1hility** [reply]
@imishin1980 if you installed via MSI, and it completed successfully, they should be in c:\Program Files\RedHat\Podman. Do you not see them there?
